projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts

```diff
@@ -84,6 +84,10 @@ export class DisplayEditComponent implements OnInit {
     // string used as class name to add add to value-component element on hover
     backgroundColor = '';
 
+    datePrecisions: 'era' | 'calendar' | 'all';
+
+    showDateLabels = false;
```

Comment:
Do we need an initial value if this is in `ngOnInit `?

Reply:
I wasn't exactly sure how this would be implemented in the end as I think these values should be chosen by the developer using the lib instead of hardcoded in `ngOnInit`, perhaps they could be `@Input`s?